data/dummy.json

```
@@ -0,0 +1,140 @@
+[
+  {
+    "id": 1,
+    "text": "What does HTML stand for?",
+    "type": "radio",
+    "answers": [
+      {
+        "id": 1,
+        "text": "Hyper text markup language",
+        "isCorrect": "true"
```

```suggestion
        "isCorrect": true
```

Since `"true"` is in quotes, it is a string. We want it to be a boolean, and in JSON a boolean does not have strings. Let's change all `isCorrect` values to booleans

---

@abdulkadiret Just a friendly reminder to do this one last change :)